.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  workflow_call:
+    inputs:
+      cli-name:
+        required: true
+        type: string
+name: Release
+jobs:
+  release:
+    permissions:
+      id-token: write
```

Comment:
I think this line is not necessary anymore, because you don't use the OIDC issuer for signing in this PR.

It might make sense to remove this line to reduce blast radius. Right now it exposes the Github OIDC token for nothing.

CC: @trishankatdatadog 

Sorry for coming up with this after the merge. I spent the last week with finishing my master degree in CS :partying_face:

Reply:
> It might make sense to remove this line to reduce blast radius. Right now it exposes the Github OIDC token for nothing.

Good catch! @rdimitrov can we fix this ASAP?

> Sorry for coming up with this after the merge. I spent the last week with finishing my master degree in CS 🥳

Congratulations! ❤️ 🚀

Reply:
@trishankatdatadog - Got it - https://github.com/theupdateframework/go-tuf/pull/255 👍